modules/survey_accounts/help/survey_accounts.md

```diff
@@ -1,43 +1,20 @@
-## Purpose
+This module is used to create and manage survey forms that are sent to participants in a study.
 
-The survey accounts module is intended to manage the surveys which
-are sent to participants in a study. It is used to generate a survey
-key, and (optionally) automatically send it to the participant, if
-their email address is provided.
+Each survey form is generated with a unique URL. This survey link URL can be emailed directly to respondents for survey completion from home, or can be used in the clinic to capture data directly from participants, e.g. on a tablet in the waiting room during a study visit. 
 
-## Intended Users
+Use the Selection Filters to search for existing surveys by PSCID, Visit Label, Instrument, or the email of a participant who has already been sent a survey form. You can use any selection of filters needed. Click **Clear Filters** to clear the current selection of filters. Click **Show Data** to display the results in a data table. 
 
-The survey accounts module is primarily used by study coordinators
-in order to generate a survey key to send to participants.
+The resulting table will display your results, which includes a unique URL for each survey generated. Click on this link to access the online survey form. This data table can be sorted by clicking on any column header (ascending and descending). 
 
-## Scope
+The last column of the table, *Status*, tells you whether the survey has been accessed. This column will display one of the following options:
 
-The survey accounts module only generates new survey keys. It does
-*not* collect the data for the surveys, which are written as LORIS
-instruments and collected via a different survey.php script in the
-LORIS `htdocs` directory.
+* *Created*: Survey was created. This is the default status once a survey is created using the "Add Survey" page.<br>
+* *Sent*: Survey was created, and an email with the unique survey URL was sent to the survey respondent. This is the default status once a survey is created and sent by email within LORIS.<br>
+* *In Progress*: Survey was opened and accessed by anyone with the link, including study personnel (not just the intended respondent).<br>
+* *Complete*: Survey was completed and submitted. After this stage, the respondent will not be able to go back and change any entries. No data will be visible via the URL once a survey is completed: clicking on the URL will display a page with the message *"Data has already been submitted"*.
 
-## Permissions
+You can create a new survey manually with data that hasn’t yet been entered into LORIS. Click **Add Survey**, which takes you to a new page. Enter the required information and click **Create Survey** to simply generate a URL (which you can send to the candidate), or **Email Survey** to automatically send the URL to the candidate. If you choose Email Survey, you’ll be able to customize the content of the email in a pop-up window (which will contain an editable generic message). 
```

Comment:
`You can create a new survey manually with data that hasn't yet been ` 
This phrase sounds a little ambiguous to me re `create` (a new instrument?) and `data` (like participant's email, which is considered metadata but not survey data). 

I'm not 100% sure what might sound clearer here, let's come back to this together.

Reply:
Revisiting - I must be misunderstanding this function. Making a note here so we can chat about it.

Reply:
looks fine - i just re-reviewed and can't think of a better wording...